website/pages/api-docs/plugins.mdx

```diff
@@ -0,0 +1,146 @@
+---
+layout: api
+page_title: Plugins - HTTP API
+sidebar_title: Plugins
+description: The `/plugin` endpoints are used to query for and interact with dynamic plugins.
+---
+
+# Plugins HTTP API
+
+The `/plugin` endpoints are used to query for and interact with
+dynamic plugins. Currently only Container Storage Interface (CSI)
+plugins are dynamic.
+
+## List Plugins
+
+This endpoint lists all dynamic plugins.
+
+| Method | Path          | Produces           |
+| ------ | ------------- | ------------------ |
+| `GET`  | `/v1/plugins` | `application/json` |
+
+The table below shows this endpoint's support for
+[blocking queries](/api-docs#blocking-queries) and
+[required ACLs](/api-docs#acls).
+
+| Blocking Queries | ACL Required                |
+| ---------------- | --------------------------- |
+| `YES`            | `namespace:csi-list-plugin` |
+
+### Parameters
+
+- `type` `(string: "")`- Specifies a string to filter volumes based on
```

Comment:
Not sure how to word it, but `type` works like a search parameter rather than a filter, it selects records of type. If it's not provided, the result list is empty. This is the same for volumes.

Reply:
Fixed.

Also the wording here refers to volumes instead of plugins! Fixed that too.